### PR TITLE
emeritus approver

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
/sig node
/assign @mrunalp

### CODE REVIEW

File: sig-node/sig-node-contributor-ladder.md [hunk 1]
```diff
@@ -114,3 +114,17 @@ There are different ways to earn trust:
...
+
+The [emeritus_approvers](https://www.kubernetes.dev/docs/guide/owners/#emeritus) section is used
+to list approvers who may no longer have time to spend on the project regularly. Keeping the list of
+active approvers up to date helps contributors find the approver for their work easier. However, people
+listed in this section are recognized for their domain knowledge and expertise.
+
+As the process of becoming a SIG Node approver is a multi-year journey, the SIG Node recognizes
+that the change of jobs and focus is only natural. So when considering the contributions to become SIG Node approver,
+the SIG Node recognizes contributions over years, no matter how long ago.
+It also makes the process of returning the emeritus approver back to regular approver easy.
```
[the remark this answers is not in the document]
Comment (SergeyKanzhelev) [reply sig-node/sig-node-contributor-ladder.md:129]: adopted

File: sig-node/sig-node-contributor-ladder.md [hunk 2]
```diff
@@ -114,3 +114,29 @@ There are different ways to earn trust:
 
 **Be present**
 Participate in SIG Node meetings by speaking about KEPs or improvements driven, or find some other way to prove the identity behind GitHub handle.
+
+## Emeritus Approvers
+
+The [emeritus_approvers](https://www.kubernetes.dev/docs/guide/owners/#emeritus) section is used
+to list approvers who may no longer have time to spend on the project regularly. Keeping the list of
+_active_ approvers up to date helps contributors find the approver for their work easier.
+However, the emeritus section as important, as people listed in it are recognized
```
Comment (haircommander) [inline sig-node/sig-node-contributor-ladder.md:123]: `However, the emeritus section is similarly important`* ?

File: sig-node/sig-node-contributor-ladder.md [hunk 3]
```diff
@@ -114,3 +114,29 @@ There are different ways to earn trust:
 
 **Be present**
 Participate in SIG Node meetings by speaking about KEPs or improvements driven, or find some other way to prove the identity behind GitHub handle.
+
+## Emeritus Approvers
+
+The [emeritus_approvers](https://www.kubernetes.dev/docs/guide/owners/#emeritus) section is used
+to list approvers who may no longer have time to spend on the project regularly. Keeping the list of
+_active_ approvers up to date helps contributors find the approver for their work easier.
```
Comment (haircommander) [inline sig-node/sig-node-contributor-ladder.md:122]: WDYT of 
`approvers up to date helps contributors find an approver for their work`

**Comment by haircommander** [comment]
a couple of nits, otherwise LGTM

**Pull request merged**